src/modules/commander/HealthAndArmingChecks/HealthAndArmingChecks.cpp

```diff
@@ -55,7 +55,9 @@ bool HealthAndArmingChecks::update(bool force_reporting, bool is_arming_request)
 {
 	_reporter.reset();
 
-	_reporter.prepare(_context.status().vehicle_type);
+	const uint8_t vehicle_type = _context.status().in_transition_mode ? vehicle_status_s::VEHICLE_TYPE_FIXED_WING :
```

Comment:
This will at least need a comment, as we're changing the meaning of `vehicle_type` (by definition in the VehicleStatus.msg it's only in FIXED_WING once outside of transition).

Reply:
done